compiler/ballerina-lang/spotbugs-exclude.xml

```diff
@@ -208,6 +208,9 @@
     <Match>
         <Class name="org.wso2.ballerinalang.compiler.bir.optimizer.BIROptimizer"/>
     </Match>
+    <Match>
+        <Class name="org.wso2.ballerinalang.compiler.bir.optimizer.BIRRecordValueOptimizer"/>
+    </Match>
```

Comment:
It is better if we fix the errors spotbugs point out and remove these lines from here.

Reply:
The spot-bug task shows a `BC_UNCONFIRMED_CAST_OF_RETURN_VALUE` error which is due to a `RecordType` casting after checking the type tag. So the warning can be ignored here. I added the `bug_pattern` to only ignore such warnings.